Library/Formula/libfreenect2.rb

```diff
@@ -0,0 +1,42 @@
+class Libfreenect2 < Formula
+  desc "Drivers and libraries for the Xbox One Kinect 2 device"
+  homepage "https://github.com/OpenKinect/libfreenect2"
+  url "https://github.com/OpenKinect/libfreenect2/archive/v0.1.1.tar.gz"
+  sha256 "4fcad0627468bedd237711b48b70a997ee8dcf30e0f1ecf9e882a0200ce647bf"
+
+  head "https://github.com/OpenKinect/libfreenect2.git"
+
+  option :universal
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "libusb"
+  depends_on "jpeg-turbo"
+  depends_on "homebrew/versions/glfw3"
```

Comment:
A formula in `Homebrew` core can not have `depends_on` for a formula in a `Tap`, unless, it is an `:optional` dependency.